README.md

```diff
@@ -25,10 +25,12 @@ The specific attributes of the robot you build will depend slightly on the type
 | Maximum 90 deg vertical scale             | 12 [in]                  | 30.48 [cm]             | 
 | Maximum height differential between sides | 14 [in]                  | 35.56 [cm]             |
 | Communication (in this guide)             | Bluetooth app (Android only) and Xbox Controller| |
-| Cost                                      | ~ $2,500                 |                        |
+| Cost (*)                                  | ~ $2,500                 |                        |
 
 Again, the above statistics depend on which components you select when buying parts. One potential change is for the motors; you can, for example, select higher RPM motors (to drive your rover faster) at the sacrifice of max stall torque, which would potentially limit your rover's ability to climb. A selection of motors that would integrate easily with the rest of the suggested rover design can be found at [GoBilda](https://www.gobilda.com/5202-series-yellow-jacket-planetary-gear-motors/).
 
+(*) Other open-source, cheaper alternatives exist but are slower, less strong, and are more fragile. See [Additiol Projects](##Additional Projects!).
```

Comment:
Additional misspelled, and the link doesn't work. I'm not 100% sure, but I think it needs to be 

[Additional Projects](##additional-projects)

Looks like links need to be all lowercase, stripped of punctuation, and hyphens for spaces.